barretenberg/cpp/src/barretenberg/vm2/simulation/context.hpp

```diff
@@ -40,7 +40,9 @@ class ContextInterface {
     // Environment.
     virtual const AztecAddress& get_address() const = 0;
     virtual const AztecAddress& get_msg_sender() const = 0;
+    virtual FF get_transaction_fee() const = 0;
```

Comment:
This can be a `const FF&`. It's more annoying for testing, but it's in line with get_address etc